scripts/upgrade

```diff
@@ -1,77 +1,154 @@
 #!/bin/bash
 
-# Exit on command errors and treat unset variables as an error
-set -eu
+#=================================================
+# GENERIC START
+#=================================================
+# IMPORT GENERIC HELPERS
+#=================================================
+
+source _common.sh
+source /usr/share/yunohost/helpers
+
+#=================================================
+# LOAD SETTINGS
+#=================================================
+ynh_script_progression --message="Loading installation settings..." --time --weight=1
 
-# See comments in install script
 app=$YNH_APP_INSTANCE_NAME
 final_path=/opt/yunohost/$app
```

Comment:
```suggestion
final_path=$(ynh_app_setting_get --app=$app --key=final_path)
```

Reply:
Yup, found about it while testing. Thanks anyway :)